### PR TITLE
Redesign nav

### PR DESCRIPTION
**Pull request by Pavlo-Tymoshchuk**
Redesign nav.
Little bit updated mobile version (design) with sub menu and added button for open sub menu.
Added component "ButtonLikeLink" in Link.
+ Waiting new logo in footer-redesign

### CODE REVIEW

File: src/components/modules/Header/Header.tsx [hunk 1]
```diff
@@ -1,18 +1,50 @@
...
+    }, [showBurgerMenu]);
+
     return (
-        <header className={styles.header}>
+        <header className={getClassnames([styles.header, showBurgerMenu && styles.headerShowBurger])}>
             <div className={styles.wrapper}>
-                <Logo redirectToHome />
-                <Navigation />
+                <div className={styles.logoWrapper}>
+                    <Logo redirectToHome />
```
Comment (karinabarinova) [inline src/components/modules/Header/Header.tsx:34]: Please update the Logo according to the design

File: src/components/modules/Header/helpers/const.ts [hunk 2]
```diff
@@ -0,0 +1,52 @@
+// Navigation links
+const LINK_HOME = "/";
+const LINK_THE_PLATFORM = "#the-platform";
+const LINK_OUR_TESTS = "#about-us";
```
Comment (karinabarinova) [inline src/components/modules/Header/helpers/const.ts:4]: Shouldn't it point to the test page?

File: src/components/modules/Header/components/Navigation.module.scss [hunk 3]
```diff
@@ -0,0 +1,152 @@
+@import "@sass/_colors.scss";
+@import "@sass/_variables.scss";
+@import "@sass/_media.scss";
+
+.navigationList {
+    display: flex;
+    gap: 1.5rem;
+
```
Comment (karinabarinova) [inline src/components/modules/Header/components/Navigation.module.scss:8]:
```suggestion
    white-space: nowrap;
```

File: src/components/modules/Header/components/Auth.module.scss [hunk 4]
```diff
@@ -0,0 +1,17 @@
+@import "@sass/_colors.scss";
+@import "@sass/_variables.scss";
+@import "@sass/_media.scss";
+
+
+.navigationList {
+    display: flex;
+    gap: 2.5rem;
+}
```
Comment (karinabarinova) [inline src/components/modules/Header/components/Auth.module.scss:9]:
```suggestion
    white-space: nowrap;
}
```

File: src/components/modules/Header/Header.tsx [hunk 5]
```diff
@@ -1,18 +1,50 @@
...
+                </div>
+                <Navigation actionToCloseBurgerMenu={actionToCloseBurgerMenu} />
                 <Auth />
             </div>
-            <MobileMenu />
+            <div className={styles.mobileWarpper}>
+                <Logo redirectToHome />
```
Comment (karinabarinova) [inline src/components/modules/Header/Header.tsx:40]: The Logo needs to be updated

File: src/components/elements/Link/ButtonLikeLink.types.ts [hunk 6]
```diff
@@ -0,0 +1,11 @@
+import { ReactNode } from "react";
+
+export type ButtonLikeLinkTypes = {
+    withArrow?: "left" | "right";
+    withBorder?: boolean;
+    children: ReactNode;
+    className?: string;
+    classNameText?: string;
+    color?: "white";
```
Comment (karinabarinova) [inline src/components/elements/Link/ButtonLikeLink.types.ts:9]: Is this added for a possible dark theme implementation? 
I can see that we don't have the styles for white color option.

**Review by karinabarinova — CHANGES_REQUESTED** [review]

File: src/components/modules/Header/components/Auth.tsx [hunk 7]
```diff
@@ -1,8 +1,9 @@
 import { Link } from "@elements/Link";
 import { formatLink } from "@utils/formatLink";
 
-import { AUTH_ITEMS } from "../helpers/const";
 import styles from "./Auth.module.scss";
+// eslint-disable-next-line import/order
+import { AUTH_ITEMS } from "../helpers/const";
```
Comment (kas1qqqq) [inline src/components/modules/Header/components/Auth.tsx:6]: reorder imports fitted rules

File: src/components/modules/Header/components/Navigation.tsx [hunk 8]
```diff
@@ -5,10 +5,12 @@ import { Link } from "@elements/Link";
...
 import { getClassnames } from "src/utils/getClassnames";
 
-import { interfaceNavigationItem, NavigationProps } from "../Header.types";
-import { NAVIGATION_ITEMS } from "../helpers/const";
 import styles from "./Navigation.module.scss";
 import { NavigationSubBlock } from "./NavigationSubBlock";
+// eslint-disable-next-line import/order
+import type { interfaceNavigationItem, NavigationProps } from "../Header.types";
+// eslint-disable-next-line import/order
+import { NAVIGATION_ITEMS } from "../helpers/const";
```
Comment (kas1qqqq) [inline src/components/modules/Header/components/Navigation.tsx:13]: reorder imports fitted rules

File: src/components/modules/Header/components/NavigationSubBlock.tsx [hunk 9]
```diff
@@ -5,8 +5,9 @@ import { ButtonLikeLink, Link } from "@elements/Link";
 import { formatLink } from "@utils/formatLink";
 import { getClassnames } from "src/utils/getClassnames";
 
-import { NavigationItem, NavigationProps } from "../Header.types";
 import styles from "./Navigation.module.scss";
+// eslint-disable-next-line import/order
+import { NavigationItem, NavigationProps } from "../Header.types";
```
Comment (kas1qqqq) [inline src/components/modules/Header/components/NavigationSubBlock.tsx:10]: the same trouble)

**Review by kas1qqqq — CHANGES_REQUESTED** [review]

File: src/components/modules/Header/Header.tsx (hunk 5 above)
Comment (Pavlo-Tymoshchuk) [reply src/components/modules/Header/Header.tsx:42]: Updated.

**Review by karinabarinova — APPROVED** [review]

**Review by Flowerinno — APPROVED** [review]
lgtm

**Review by kas1qqqq — APPROVED** [review]

**Pull request merged**